ios/eMobility.xcodeproj/project.pbxproj

```diff
@@ -921,6 +922,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LIBRARY_SEARCH_PATHS = (
+					"$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)",
```

Comment:
Yes it is. Yes, it was because of an error on the build and the doc of stripe said to remove those lines to fix it. It has worked but it seems it is no longer needed

Reply:
Could you please check if similar changes done are not necessary with the latest strip lib?  

I do not really like the iOS version bump to 11 in the Podile for example. 

Thanks.

Reply:
The bump is said to be required in their doc

Reply:
@jerome-benoit , do you think we can upgrade XCode to 12.5 on the pipeline ?